operators/advanced_network/advanced_network/managers/gpunetio/adv_network_doca_mgr.h

```diff
@@ -45,8 +45,8 @@
 #include <doca_pe.h>
 #include <doca_eth_txq_gpu_data_path.h>
 
-#include "adv_network_mgr.h"
-#include "adv_network_common.h"
+#include "../../manager.h"
+#include "../../common.h"
```

Comment:
Thoughts on relative pathing versus `target_include_directories` to include parent directories?

Reply:
Either really, this is private headers so there is not question of portability post-install. I went relative for the same reason as [here](https://github.com/nvidia-holoscan/holohub/pull/711#discussion_r1976020134). Totally fine to do `"advanced_network/common.h"` instead

Reply:
Same comment as below